packages/form-field/src/FormFieldInputContainer/FormFieldInputContainer.styles.ts

```diff
@@ -60,7 +61,7 @@ export const inputWrapperModeStyles: Record<Theme, string> = {
 
     &:hover,
     &:active {
-      &:not([aria-disabled='true']):not(:focus) {
+      &:not(:focus) {
```

Comment:
I moved the `aria-disabled` attribute from the container div to the input div. In doing so, I needed to refactor these styles

I updated the styles under `inputWrapperModeStyles` to always be applied (no longer checking for `:not([aria-disabled='true'])`). I overwrite or undo the styles in the `inputWrapperDisabledStyles` by only applying the classname if `disabled` prop is true